plugins/auth-backend/src/identity/StaticKeyStore.test.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { promises as fs } from 'fs';
+import { StaticKeyStore } from './StaticKeyStore';
+import { AnyJWK } from './types';
+import { ConfigReader } from '@backstage/core-app-api';
+
+const publicKeyPath = '/mnt/public.pem';
+const privateKeyPath = '/mnt/private.pem';
+const privateKey = `
+-----BEGIN PRIVATE KEY-----
+MIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgR8Ja2ppMEgOm1KeY
+Kpje00U1luybndt6yC263vcgeKqhRANCAAS+slUrS9JXgtHB1RcDnmlveuu4H3Zm
+hQRjvYdO+Mg/3FJss6FaExESTzhPSr3X+be/exarkTMchbDXNEdCKwpn
+-----END PRIVATE KEY-----
+`.trim();
+
+const publicKey = `
+-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEvrJVK0vSV4LRwdUXA55pb3rruB92
+ZoUEY72HTvjIP9xSbLOhWhMREk84T0q91/m3v3sWq5EzHIWw1zRHQisKZw==
+-----END PUBLIC KEY-----
+`.trim();
+
+const config = new ConfigReader({
+  keys: [
+    {
+      publicKeyFile: publicKeyPath,
+      privateKeyFile: privateKeyPath,
+      keyId: '1',
+      algorithm: 'ES256',
+    },
+    {
+      publicKeyFile: publicKeyPath,
+      privateKeyFile: privateKeyPath,
+      keyId: '2',
+      algorithm: 'ES256',
+    },
+  ],
+});
+
+jest.mock('fs/promises');
```

Comment:
Consider using `createMockDirectory` ([example here](https://github.com/backstage/backstage/blob/c1df31c72411b85623fbaa39d9290ee6a3c64118/packages/backend-common/src/reading/tree/TarArchiveResponse.test.ts#L27)) to put actual files in a temp folder and point to those instead.